src/iOS.Core/Utilities/UISearchBarExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using UIKit;
+
+namespace Bit.iOS.Core.Utilities
+{
+    public static class UISearchBarExtensions
+    {
+        public static void ApplyDarkThemesColors(this UISearchBar searchBar)
+        {
+            var versionParts = UIDevice.CurrentDevice.SystemVersion.Split('.');
+            if (versionParts.Length > 0 && int.TryParse(versionParts[0], out var version))
+            {
+                if (version >= 13)
```

Comment:
This can be done easier by doing: `if (UIDevice.CurrentDevice.CheckSystemVersion(13, 0))`